atspi-common/src/events/object.rs

```diff
@@ -1019,7 +1085,14 @@ impl TryFrom<&zbus::Message> for ObjectEvents {
 			"ColumnDeleted" => Ok(ObjectEvents::ColumnDeleted(ev.try_into()?)),
 			"TextBoundsChanged" => Ok(ObjectEvents::TextBoundsChanged(ev.try_into()?)),
 			"TextSelectionChanged" => Ok(ObjectEvents::TextSelectionChanged(ev.try_into()?)),
-			"TextChanged" => Ok(ObjectEvents::TextChanged(ev.try_into()?)),
+			"TextChanged" => {
+				let body: EventBodyOwned = ev.try_into()?;
+				match body.kind.as_str() {
```

Comment:
As above, but for TextChanged, the discriminants may be 'inserted' and 'removed'?